pkg/controller/utils/delayingreconciler.go

```diff
@@ -0,0 +1,39 @@
+package utils
+
+import (
+	"context"
+	log "github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"time"
+)
+
+const requeueDelay = time.Minute * 2
+
+type delayingReconciler struct {
+	inner  reconcile.Reconciler
```

Comment:
NIT: I think a name like `wrappedReconciler` (to copy from [the metrics reconciler wrapper](https://github.com/openshift/aws-account-operator/blob/557c6d33db8d4b5eb9fd19b873c2b7e2d0f730e9/pkg/controller/utils/reconcilermetrics.go)) would be a better name for this than `inner`.